TASVideos/Services/DevFallbackFileProvider.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Microsoft.Extensions.FileProviders;
+using Microsoft.Extensions.Primitives;
+
+namespace TASVideos.Services;
+
+public class DevFallbackFileProvider(IFileProvider originalProvider) : IFileProvider
+{
+	public IDirectoryContents GetDirectoryContents(string subpath)
+	{
+		return originalProvider.GetDirectoryContents(subpath);
+	}
+
+	public IFileInfo GetFileInfo(string subpath)
+	{
+		IFileInfo fileInfo = originalProvider.GetFileInfo(subpath);
+		if (!fileInfo.Exists && subpath.StartsWith("/media/"))
+		{
+			return originalProvider.GetFileInfo("/images/tasvideos_rss.png");
```

Comment:
Memoise?

Reply:
Do you mean we should cache the result of the hard-coded fallback file to avoid calling the method again and again?
To that I answer: Wouldn't caching be the job of the provider's GetFileInfo? If it doesn't do it, why should we? It would certainly cause issues if people would manually move around image files or replace them.